code/game/objects/items/weapons/traps.dm

```diff
@@ -157,8 +157,17 @@ Freeing yourself is much harder than freeing someone else. Calling for help is a
 	update_icon()
 	STOP_PROCESSING(SSobj, src)
 
-
-
+//Attempting to resist out of a beartrap will not work, and you'll get nothing but pain for trying
+/obj/item/weapon/beartrap/resist_buckle(var/mob/user)
+	if (user == buckled_mob && !user.stunned)
+		//We check stunned here, and a failure stuns the victim. This prevents someone from just spam-resisting and instantly killing themselves
+		if (user.client)
+			fail_attempt(user)
+			to_chat(user, SPAN_WARNING("Struggling out of this isn't going to work, you'll need to try to release \the [src] with your hands or a tool"))
```

Comment:
Would it be possible for resisting out to activate the proc for escaping with your hands, or with a tool if you're holding one?

Reply:
It would be entirely possible, i chose not to for a reason.

traps are supposed to be a hugely inconvenient, annoying, and harrowing experience. Their raison d'etre is to provide a chilling effect and force you to rethink your course of action. This starts with the process of escaping it. Attempting to resist is immediately punished, you have to find the little sprite and click on it